media/js/firefox/new/desktop/exp.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function (Mozilla) {
+    'use strict';
+
+    var href = window.location.href;
+    var initTrafficCop = function () {
+        if (href.indexOf('v=') !== -1) {
+            Mozilla.StubAttribution.experimentName = 'experiment_new-redesign';
+            /* update dataLayer with experiment info */
+            if (href.indexOf('v=a') !== -1) {
+                Mozilla.StubAttribution.experimentVariation = 'va';
+                window.dataLayer.push({
+                    'data-ex-variant': 'new-redesign-control',
+                    'data-ex-name': 'new-redesign-Experiment'
```

Comment:
On the CRO team, we have the convention of adding "CRO" at the beginning of `data-ex-name` --so at a glance it can be attributed to a particular team.
Example: `'data-ex-name': 'CRO-Whatsnew-77-Experiment'`

Reply:
Added NABrand- to our name 👍